sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/http/models/ResponseBodyHandling.java

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.generic.core.http.models;
+
+import com.generic.core.annotation.ServiceInterface;
+
+/**
+ * Enum that defines how to handle the body of an HTTP response.
+ */
+public enum ResponseBodyHandling {
```

Comment:
We need to find a better name for this enum. Something like `ResponseBodyMode` or `ResponseMode` might be better?

Reply:
Is `ResponseBodyHandlingMode` too much?